src/components/DatePicker.js

```diff
@@ -28,6 +28,9 @@ class CustomInput extends PureComponent {
       readonly: PropTypes.bool,
       placeholder: PropTypes.string,
       size: PropTypes.string.isRequired,
+      popupProps: PropTypes.shape({
```

Comment:
Pls give me a code snippet of how do you use this `popUpProps`

Reply:
![image](https://user-images.githubusercontent.com/52645649/88874510-d1aa3c00-d251-11ea-9789-ce4e9c283bc5.png)
This includes both sublabelProps and popupProps